src/runaqa.ts

```diff
@@ -312,13 +359,129 @@ async function runGetSh(
   }
 }
 
+/**
+ * Sets up enviroment to generate parallelList.mk
+ * @param  {string} version JDK Version being tested
+ * @param  {string} jdksource Source for JDK
+ * @param  {[string]} customizedSdkUrl Download link for JDK binaries
+ * @param  {[string]} sdkdir Directory for SDK
+ * @param  {[string]} buildList AQAvit Test suite
+ * @param  {[string]} aqatestsRepo Alternative aqatestRepo
+ * @param  {[string]} openj9Repo Alternative openj9Repo
+ * @param  {[string]} tkgRepo Alternative TKG repo
+ * @param  {[string]} vendorTestParams Vendor provided test parameters
+ * @param  {[string]} aqasystemtestsRepo Alternative AQA-systemtestRepo
+ * @return {[null]}  null
+ */
+export async function setupParallelEnv(
+  version: string,
+  jdksource: string,
+  customizedSdkUrl: string,
+  sdkdir: string,
+  buildList: string,
+  aqatestsRepo: string,
+  openj9Repo: string,
+  tkgRepo: string,
+  vendorTestParams: string,
+  aqasystemtestsRepo: string,
+  numMachines: string
+): Promise<void> {
+
+  await setupTestEnv(version, jdksource, customizedSdkUrl, sdkdir, buildList, aqatestsRepo, openj9Repo, tkgRepo, vendorTestParams, aqasystemtestsRepo);
+  process.chdir('TKG');
+  process.env.PARALLEL_OPTIONS = `PARALLEL_OPTIONS=TEST=${buildList} TEST_TIME= NUM_MACHINES=${numMachines}`;
```

Comment:
TEST should equal to the test target.

Reply:
I believe `BUILD_LIST` here would be the target? Since the values of `BUILD_LIST` would be one of `openjdk, functional, system, perf`. Or am I mistaken? If the verbiage does not make sense that could be adjusted. However, I believe that this value matches what is set in the env variable `BUILD_LIST`.

Reply:
`BUILD_LIST` is not the target, it is the directory to be compiled. Some target and `BUILD_LIST` happen to share the same name, for example `functional`, but target also could be `sanity.functional` which is different from BUILD_LIST

Reply:
Thank you for the explanation. I will update the code.